setup.py

```diff
@@ -1,37 +1,5 @@
-try:
-    from setuptools import setup, find_packages
-except ImportError:
-    from distutils.core import setup
+from pathlib import Path
+from setuptools import setup
 
 
-def get_requirements():
-    with open("./requirements.txt") as file:
-        return file.readlines()
-
-
-version = {}
-with open("./release_bot/version.py") as fp:
-    exec(fp.read(), version)
-
-setup(
-    name='release-bot',
-    version=version["__version__"],
-    packages=['release_bot'],
-    python_requires='>=3.6',
-    url='https://github.com/user-cont/release-bot',
-    license='GPLv3+',
-    author='Red Hat',
-    author_email='user-cont@redhat.com',
-    description='Automated releasing from GitHub repositories',
-    install_requires=get_requirements(),
-    classifiers=[
-        'Development Status :: 3 - Alpha',
-        'Intended Audience :: Developers',
-        'Topic :: Software Development',
-        'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',
-        'Programming Language :: Python :: 3'
-    ],
-    entry_points={'console_scripts': [
-        'release-bot=release_bot.releasebot:main',
-    ]}
-)
+setup(install_requires=Path("./requirements.txt").read_text(), )
```

Comment:
I wonder if the `file` directive could also be used for this

Reply:
Yeah, I tried that, but it doesn't work

```ini
[options]
install_requires = file: requirements.txt
```
results in

```python
pkg_resources.RequirementParseError: Invalid requirement, parse error at "': requir'"
```

and also [doc](https://setuptools.readthedocs.io/en/latest/setuptools.html#options) does not mention `file:` to be used with `install_requires`.